core/src/main/scala/org/bitcoins/core/protocol/blockchain/ChainParams.scala

```diff
@@ -48,6 +42,10 @@ sealed abstract class ChainParams {
   /** The Genesis [[org.bitcoins.core.protocol.blockchain.Block Block]] in the blockchain. */
   def genesisBlock: Block
 
+  def chainHash: DoubleSha256Digest = genesisBlock.blockHeader.hash
```

Comment:
Can we call it `genesisHash`? `chainHash` is a bit ambiguous as a blockchain as a bunch of hashes